src/test/java/org/opentripplanner/updater/trip/RealtimeTestEnvironment.java

```diff
@@ -256,15 +264,22 @@ public UpdateResult applyEstimatedTimetable(List<EstimatedTimetableDeliveryStruc
   // GTFS-RT updates
 
   public UpdateResult applyTripUpdate(GtfsRealtime.TripUpdate update) {
-    return applyTripUpdates(List.of(update));
+    return applyTripUpdates(List.of(update), false);
+  }
+
+  public UpdateResult applyTripUpdate(GtfsRealtime.TripUpdate update, boolean differential) {
```

Comment:
Can you convert the boolean to an enum making it clear what exactly that means? After some discussion in another PR we concluded that we want to change to an enum in the main code as well. That I will do in a separate PR.

Reply:
To avoid conflicts, maybe it's easier if this gets either updated in this pr or in your pr depending on what gets merged first.

Reply:
Yes, lets do that.